include/LD/assets/styles/override_lds.css

```diff
@@ -158,12 +158,6 @@ hr {
     top: -0.0625rem;
     right: -0.0625rem;
     animation-name: dotsBounceAfter-extraExtraSmall; }
-  .slds-masseditprogress {
```

Comment:
Do you have to eliminate this class? It is being used by the mass edit popup information screen. If you delete it that will probably break. Can you find another way of styling your objects that does not conflict with this class?

Reply:
I think this not affect anything as i know ,if you know where affect and what affect i can fix normaly.